components/new_channel_flow/new_channel_flow.tsx

```diff
@@ -116,17 +141,27 @@ export default class NewChannelFlow extends React.Component {
             return;
         }
 
-        const {actions, currentTeamId} = this.props;
-        const channel = {
+        const {actions, currentUserId, currentTeamId} = this.props;
+        const channel: Channel = {
             team_id: currentTeamId,
             name: this.state.channelName,
             display_name: this.state.channelDisplayName,
             purpose: this.state.channelPurpose,
             header: this.state.channelHeader,
             type: this.state.channelType,
+            create_at: 0,
+            creator_id: '',
+            delete_at: 0,
+            extra_update_at: 0,
+            group_constrained: false,
+            id: '',
+            last_post_at: 0,
+            scheme_id: '',
+            total_msg_count: 0,
+            update_at: 0,
         };
 
-        actions.createChannel(channel).then((result) => {
+        actions.createChannel(channel, currentUserId).then((result: {data: any; error?: any}) => {
```

Comment:
Can we avoid 'any' here? Maybe there's appropriate type returned from createChannel in redux?

Reply:
Done :+1: